monitor/monitor.go

```diff
@@ -255,3 +276,10 @@ type nopCloser struct {
 }
 
 func (c nopCloser) Close() error { return nil }
+
+func defaultShell(os string) []string {
```

Comment:
Is this what's used to run in the container?

- Should this ideally be shared with what's defined in [BuildKit](https://github.com/moby/buildkit/blob/9b28f40b8e80244e27cc2935c1522f2f63e7f55e/frontend/dockerfile/dockerfile2llb/defaultshell.go#L3-L8)?
- Also curious what should happen if the image actually has a different shell configured (`SHELL`), and perhaps expects something different than the default

Reply:
This isn't inside the container - that's handled separately.

This is to just run something on the client system, outside of the debugging session.

Reply:
Ah, yeah, if it's not inside the container then definitely doesn't need to be shared 👍